chat-android/src/main/java/com/ably/chat/Utils.kt

```diff
@@ -50,6 +55,27 @@ suspend fun Channel.publishCoroutine(message: PubSubMessage) = suspendCoroutine
     )
 }
 
+suspend fun PubSubPresence.getCoroutine(param: Param) = withContext(Dispatchers.IO) {
+    get(param)
+}
+
+@SuppressWarnings("SpreadOperator")
+suspend fun PubSubPresence.getCoroutine(params: List<Param>) = withContext(Dispatchers.IO) {
+    get(*params.toTypedArray())
+}
+
+suspend fun PubSubPresence.enterClientCoroutine(clientId: String, data: JsonElement?) = withContext(Dispatchers.IO) {
+    enterClient(clientId, data)
```

Comment:
We can provide `CompletionListener` here to denote success/failure of the operation -> https://github.com/ably/ably-java/blob/dc4d464ed73084e1276eeccb20ba994472a112f7/lib/src/main/java/io/ably/lib/realtime/Presence.java#L583

Reply:
Not sure how this operation is different from `Attach` or `Detach`, meaning we don't have to run it on `Dispatchers.IO`, implementation can be same as `AttachCoroutine` or `DetachCoroutine`

Reply:
fixed